src/i18n/en-US.json

```diff
@@ -596,6 +613,11 @@
   "guestRoomToggleInfoDisabled": "You can't disable the guest option in this conversation, as it has been created by someone from another team.",
   "guestRoomToggleInfoExtended": "Open this conversation to people outside your team. You can always change it later.",
   "guestRoomToggleInfoHead": "Guest Links",
+  "guestLinkPasswordModal.headline": "[Group Conversation] \n Enter password",
```

Comment:
Is this `\n` really working 🤔 ?

Reply:
yeah if you do white-space: pre or pre-line it works.